addOns/help/src/main/javahelp/contents/ui/tabs/history.html

```diff
@@ -115,6 +115,11 @@ <H3>Manage History Tags...</H3>
 the <a href="../../start/features/tags.html">tags</a> associated with the
 request.
 
+<H3>Jump to History Id...</H3>
```

Comment:
`Id` -> `ID`? (all occurences)

Reply:
I like the full caps variant more too, but in the UI it’s mixed ☹️

Reply:
Oh, okay. I think Id is fine in code since we use camelCase everywhere but in the UI we should use ID.

Reply:
I can tackle that in another core PR and another help PR :wink:

Reply:
Cool :smiley:  :+1:

Reply:
Amazingly I searched "Id" matching case as a whole word in *.html files within the "help" project and this is the only hit.